leaks_test.go

```diff
@@ -40,17 +40,26 @@ func testOptions() Option {
 }
 
 func TestFind(t *testing.T) {
-	require.NoError(t, Find(), "Should find no leaks by default")
+	t.Run("Should find no leaks by default", func(t *testing.T) {
+		require.NoError(t, Find())
+	})
 
-	bg := startBlockedG()
-	err := Find(testOptions())
-	require.Error(t, err, "Should find leaks with leaked goroutine")
-	assert.Contains(t, err.Error(), "blockedG")
-	assert.Contains(t, err.Error(), "created by go.uber.org/goleak.startBlockedG")
-
-	// Once we unblock the goroutine, we shouldn't have leaks.
-	bg.unblock()
-	require.NoError(t, Find(), "Should find no leaks by default")
+	t.Run("Find leaks with leaked goroutine", func(t *testing.T) {
+		bg := startBlockedG()
+		err := Find(testOptions())
+		require.Error(t, err, "Should find leaks with leaked goroutine")
+		assert.Contains(t, err.Error(), "blockedG")
+		assert.Contains(t, err.Error(), "created by go.uber.org/goleak.startBlockedG")
```

Comment:
ack. This doesn't look doable rn because we're using 1.7.0 testify, and ErrorContains was introduced in 1.7.1. I'll follow up with another PR to address this.